zookeeper/src/main/java/com/linecorp/armeria/client/zookeeper/ZooKeeperEndpointGroup.java

```diff
@@ -67,58 +66,41 @@ public ZooKeeperEndpointGroup(String zkConnectionStr, String zNodePath, int sess
      */
     public ZooKeeperEndpointGroup(String zkConnectionStr, String zNodePath, int sessionTimeout,
                                   NodeValueCodec nodeValueCodec) {
-        zooKeeperConnector = new ZooKeeperConnector(zkConnectionStr, zNodePath, sessionTimeout,
-                                                    createListener());
         this.nodeValueCodec = requireNonNull(nodeValueCodec, "nodeValueCodec");
-        zooKeeperConnector.connect();
+        ExponentialBackoffRetry retryPolicy = new ExponentialBackoffRetry(1000, 3);
```

Comment:
It seems like all of the members in `ExponentialBackoffRetry` are final so it looks like a stateless instance.
This same `retryPolicy` used in three different places. How about declaring something like `defaultRetryPolicy` in one place and use it?

Reply:
How about `com.linecorp.armeria.common.zookeeper.Constants`? We can move `ZookeeperUpdatingListenerBuilder#[DEFAULT_CONNECT_TIMEOUT, DEFAULT_SESSION_TIMEOUT]` also.

Reply:
The package has to be `com.linecorp.armeria.internal.zookeeper` under `armeria-zookeeper`, because such constants should be hidden from the public API.